utils/ccdatofhir/ccda_to_fhir.bal

```diff
@@ -0,0 +1,186 @@
+                    isNamespaceAvailable = false;
+                    componentElements = xmlPayload/<component>/<structuredBody>/<component>;
+                }
+                if componentElements.length() > 0 {
+                    foreach xml:Element componentElement in componentElements {
+                        xml<xml:Element> sectionElement;
+                        if isNamespaceAvailable {
+                            sectionElement = componentElement/<v3:section>;
+                        } else {
+                            sectionElement = componentElement/<section>;
+                        }
+
+                        if sectionElement is xml:Element {
+                            xml<xml:Element> codeElement;
+                            if isNamespaceAvailable {
+                                codeElement = sectionElement/<v3:code>;
+                            } else {
+                                codeElement = sectionElement/<code>;
+                            }
+
+                            if codeElement is xml:Element {
+                                string codeVal = check codeElement.code;
+                                xml<xml:Element>|xml<xml:Comment>|xml<xml:ProcessingInstruction>|xml<xml:Text> entryElements = (isNamespaceAvailable) ? sectionElement/<v3:entry> : sectionElement/<entry>;
+                                if entryElements is xml<xml:Element> {
+                                    foreach xml:Element entryElement in entryElements {
+                                        xml<xml:Element> actElement;
+                                        xml<xml:Element> substanceAdministrationElement;
+                                        xml<xml:Element> procedureElement;
+                                        xml<xml:Element> organizerElement;
+
+                                        if isNamespaceAvailable {
+                                            actElement = entryElement.children().<v3:act>;
+                                            substanceAdministrationElement = entryElement.children().<v3:substanceAdministration>;
+                                            procedureElement = entryElement.children().<v3:procedure>;
+                                            organizerElement = entryElement.children().<v3:organizer>;
+                                        } else {
+                                            actElement = entryElement.children().<act>;
+                                            substanceAdministrationElement = entryElement.children().<substanceAdministration>;
+                                            procedureElement = entryElement.children().<procedure>;
+                                            organizerElement = entryElement.children().<organizer>;
+                                        }
+
+                                        match codeVal {
+                                            "48765-2" => {
```

Comment:
Can use constants for these values?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/817dadb5178bde7d809333a26f465c6688d3879a